### PR TITLE
Add more info for Assert message in ProcessStartInfoTests

### PR DESCRIPTION
**Pull request by Maxwe11**
https://github.com/dotnet/corefx/issues/3075#issuecomment-137559372

### CODE REVIEW

File: src/System.Diagnostics.Process/tests/ProcessStartInfoTests.cs [hunk 1]
```diff
@@ -217,7 +217,13 @@ public void TestEnvironmentOfChildProcess()
             p.WaitForExit(); // This ensures async event handlers are finished processing.
 
             // Validate against StartInfo.Environment
-            Assert.True(expectedEnv.SetEquals(actualEnv));
+            if (expectedEnv.SetEquals(actualEnv))
```
Comment (stephentoub) [inline src/System.Diagnostics.Process/tests/ProcessStartInfoTests.cs]: There should be a ```!``` in the condition, right?

Comment (Maxwe11) [inline src/System.Diagnostics.Process/tests/ProcessStartInfoTests.cs]: ops, yes you are right

**Comment by stephentoub** [comment]
LGTM

**Comment by mellinoe** [comment]
LGTM

**Pull request merged**